### PR TITLE
Add a tutorial for the advanced order types and documentation for the complex clearing

### PR DESCRIPTION
**Pull request by adamsjohanna**

### CODE REVIEW

File: assume/markets/clearing_algorithms/complex_clearing.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
+import cProfile
```
Comment (nick-harder) [inline assume/markets/clearing_algorithms/complex_clearing.py:5]: why is this needed?

File: assume/strategies/advanced_orders.py [hunk 2]
```diff
@@ -166,20 +166,21 @@ def calculate_bids(
...
+                    "only_hours": product_tuples[0][2],
+                    "price": mean_price,
+                    "volume": bid_quantity_block,
+                    "bid_type": "BB",
+                    "min_acceptance_ratio": 1,
+                    "accepted_volume": {product[0]: 0 for product in product_tuples},
+                }
+            )
 
         bids = self.remove_empty_bids(bids)
```
Comment (nick-harder) [inline assume/strategies/advanced_orders.py:185]: since there is already a check for volume != 0 this is not needed

File: examples/examples.py [hunk 3]
```diff
@@ -23,8 +23,8 @@
         "study_case": "dam_with_complex_clearing",
     },
     "small_with_BB_and_LB": {
-        "scenario": "example_01d",
-        "study_case": "dam_with_complex_clearing",
+        "scenario": "example_01c",
```
Comment (nick-harder) [inline examples/examples.py:26]: did you merge c and d? and also I saw you added a new e example, is it also in the dict of available examples now?

File: examples/examples.py [hunk 4]
```diff
@@ -64,7 +64,7 @@
     - local_db: without database and grafana
     - timescale: with database and grafana (note: you need docker installed)
     """
-    data_format = "local_db"  # "local_db" or "timescale"
+    data_format = "timescale"  # "local_db" or "timescale"
```
Comment (nick-harder) [inline examples/examples.py:67]: I would prefer to keep local db in the exmaple on the main

**Comment by nick-harder** [comment]
@adamsjohanna also the tutorial has to be added to the docs, see docs folder for details

File: assume/strategies/advanced_orders.py (hunk 2 above)
Comment (adamsjohanna) [reply assume/strategies/advanced_orders.py:185]: We only check the volume for the block order, so we need this here.

File: assume/strategies/flexable_storage.py [hunk 5]
```diff
@@ -335,15 +335,6 @@ def calculate_bids(
                 )
                 theoretic_SOC += delta_soc
                 previous_power = bid_quantity + current_power
-                time_delta = (end - start) / timedelta(hours=1)
```
[the remark this answers is not in the document]
Comment (adamsjohanna) [reply assume/strategies/flexable_storage.py:338]: This was just double

**Review by nick-harder — APPROVED** [review]
